### ISSUE TITLE
Performance Impact

### ISSUE THREAD

**Issue by Gitoffthelawn** [Quicksaver/The-Fox--Only-Better#194]
When loading 15 tabs, `about:performance` reports *Beyond Australis* as having a greater performance impact than most other extensions I tested.  Many *high-impact* events are reported.

After pages are loaded, `about:performance` reports that *Beyond Australis* has minimal impact (1% CPU usage).

I only enable the *Awesome Bar* improvements in *Beyond Australis*.  *Slim Chrome*, and hence *Sky Lights*, are both disabled.

To hopefully get the most accurate data out of `about:performance`, I use this procedure:
   1. Load Firefox
   2. Wait 30 seconds to let it settle
   3. Open `about:performance`
   4. Uncheck `Display the latest 10s`
   5. Check `Refresh Automatically`
   6. Click on bookmarks menu, select a bookmarks folder with 15 items, and select `Open all in tabs`
   7. Click on each tab once it is loaded
   8. Click on the `about:performance` tab
   9. Observe the results

**Comment by Quicksaver** [comment]
Thank you for the steps, I'll be sure to check it out and get back to you.

**Comment by Quicksaver** [comment]
I don't see any issues of the sort at all. Could you please try with [beta version 1.4.7b1](https://github.com/Quicksaver/The-Fox--Only-Better/releases/tag/v1.4.7b1) and see if it makes a difference for you?

**Comment by Gitoffthelawn** [comment]
Yes. I'm overbooked, but I'll do my best to get to it.  If you want to add a little pressure... no problem... just ping me again here.... :)

**Comment by Quicksaver** [comment]
No worries, I understand completely. Whenever you can. 😉
